src/enmerkar_underscore/__init__.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 import django
 
 import django
```

Comment:
Not related to your changes, but I think we can remove the extra import of django